src/reformatters/noaa/gfs/forecast/template_config.py

```diff
@@ -0,0 +1,635 @@
+from collections.abc import Sequence
+from typing import Any, Final, Literal
+
+import numpy as np
+import pandas as pd
+
+from reformatters.common.config_models import (
+    Coordinate,
+    CoordinateAttrs,
+    DatasetAttributes,
+    DataVarAttrs,
+    Encoding,
+    StatisticsApproximate,
+    replace,
+)
+from reformatters.common.types import DatetimeLike
+from reformatters.common.zarr import (
+    BLOSC_4BYTE_ZSTD_LEVEL3_SHUFFLE,
+    BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE,
+)
+from reformatters.noaa.gfs.gfs_config_models import GFSDataVar, GFSInternalAttrs
+
+DATASET_ID = "noaa-gfs-forecast"
+DATASET_VERSION = "0.0.0"
+
+INIT_TIME_START = pd.Timestamp("2021-04-13T00:00")
+INIT_TIME_FREQUENCY = pd.Timedelta("6h")
+
+DATASET_ATTRIBUTES = DatasetAttributes(
+    dataset_id=DATASET_ID,
+    dataset_version=DATASET_VERSION,
+    name="NOAA GFS forecast, 16 day",
```

Comment:
```suggestion
    name="NOAA GFS forecast",
```
We don't need to separate into multiple forecast archives so we can stay concise